Readme.md

```diff
@@ -25,6 +25,19 @@ If you want to set up the OpenId features without the recipe, you need to enable
 
 For testing, run your Orchard Core app first, then `Lombiq.OrchardCoreApiClient.Tester`. You may need to edit the [_Program.cs_](Lombiq.OrchardCoreApiClient.Tester/Program.cs) according to your OpenID Application's settings (`ClientId`, `ClientSecret`, and `DefaultTenantUri`) and then run the console application.
 
+### Use your own API endpoints
+
+You can add your own API endpoints by creating a new interface that's inherited from `IOrchardCoreApi` and add your own methods. For example, you can create a new interface called `IOrchardCoreApiExtended` and add a new method to it:
```

Comment:
```suggestion
You can add your own API endpoints by creating a new interface (that's inherited from `IOrchardCoreApi` if you want to access the endpoints surfaced by it too) and adding your own methods. For example, you can create a new interface called `IOrchardCoreApiExtended` and add a new method to it:
```

Reply:
Not really. TApi must be IOrchardCoreApi as the `CreateAndSetupTenantAsync` is back in the "base" class. So the TApi must implement `CreateAsync`:
```csharp
public class ApiClient<TApi> : IDisposable
    where TApi : IOrchardCoreApi
```